test/functional/test-3p-environment.js

```diff
@@ -35,6 +35,10 @@ describe('3p environment', () => {
     });
   });
 
+  afterEach(() => {
+    testWin = null;
```

Comment:
From my reading it doesn't seem like that would actually work memory-wise (but it might and one could definitely fix it). But it would not work with `--watch`, right?